File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/WebPlayerShareEntityServiceEndpoint.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class WebPlayerShareEntityServiceEndpoint{

	@SerializedName("serializedShareEntity")
	private String serializedShareEntity;

	public void setSerializedShareEntity(String serializedShareEntity){
		this.serializedShareEntity = serializedShareEntity;
	}

	public String getSerializedShareEntity(){
		return serializedShareEntity;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/WebPlayerShareEntityServiceEndpoint.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class WebPlayerShareEntityServiceEndpoint : Parcelable, Serializable {
    @SerializedName("serializedShareEntity")
    var serializedShareEntity: String? = null
    override fun toString(): String {
        return "WebPlayerShareEntityServiceEndpoint{" +
                "serializedShareEntity = '" + serializedShareEntity + '\'' +
                "}"
    }
}